codecs/png/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "oxipng",
+  "scripts": {
+    "build": "../build-rust.sh rm -rf pkg && wasm-pack build --debug --target web -- --verbose --locked && rm pkg/.gitignore"
```

Comment:
Um is this meant to still be built in a debug mode?

Reply:
Ah good catch! Definitely not.